libdrgn/symbol.c

```diff
@@ -65,3 +65,11 @@ LIBDRGN_PUBLIC bool drgn_symbol_eq(struct drgn_symbol *a, struct drgn_symbol *b)
 		a->size == b->size && a->binding == b->binding &&
 		a->kind == b->kind);
 }
+
+LIBDRGN_PUBLIC void drgn_symbols_free(struct drgn_symbol **syms, size_t count)
+{
+	for (size_t i = 0; i < count; i++)
+		if (syms[i])
+			drgn_symbol_destroy(syms[i]);
```

Comment:
`drgn_symbol_destroy()` already handles a `NULL` argument.

```suggestion
		drgn_symbol_destroy(syms[i]);
```